app/controllers/better_together/notifications_controller.rb

```diff
@@ -35,6 +35,17 @@ def mark_as_read # rubocop:todo Metrics/AbcSize, Metrics/MethodLength
         end
       end
     end
-    # rubocop:enable Metrics/MethodLength
+
+    def mark_notification_as_read(id)
+      @notification = helpers.current_person.notifications.find(id)
+      @notification.update(read_at: Time.current)
+    end
+
+    def mark_message_as_read(id)
+      @notification = helpers.current_person.notifications.unread.includes(
+        :event
+      ).references(:event).where(event: { record_id: id })
+      @notification.update(read_at: Time.current)
```

Comment:
This should probably use the pluralized variable name and call the `update_all` method. I'm unsure of the expected behaviour of calling update on a collection. Is it perhaps an alias for update_all or vice-versa?